packages/server/src/sdk/app/tables/getters.ts

```diff
@@ -20,11 +20,15 @@ import datasources from "../datasources"
 import sdk from "../../../sdk"
 
 function processInternalTables(docs: AllDocsResponse<Table[]>): Table[] {
```

Comment:
I would be tempted to update this function to match the function params of the single `processInternalTable`. The function is only used in two places, and it would be fine for those places to map out the tables first, with a `docs.rows.map(row => row.doc)` then we can update the function param to expect a list of tables instead, `tables: Table[]`.